CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ set(CMAKE_CXX_FLAGS
 	-Werror=uninitialized
 	-Werror=init-self
 	-Wcast-qual
-	-Wconversion
+	-Wno-conversion
```

Comment:
I thought that too but even 1.0f results in a complaint that I have an
implicit conversion from double to float and that I may be losing precision
from that (warning makes sense and is important but in that context it is
nonsense. Specially together with -Werror) When you build for px4 obviously
1.0f will be a float since that what is natively supported by hardware so
no problem there.

On 16 Aug 2016 18:24, "James Goppert" <notifications@github.com> wrote:

> In CMakeLists.txt
> <https://github.com/PX4/Matrix/pull/26#discussion_r74969099>:
>
> > @@ -51,7 +51,7 @@ set(CMAKE_CXX_FLAGS
> >  	-Werror=uninitialized
> >  	-Werror=init-self
> >  	-Wcast-qual
> > -	-Wconversion
> > +	-Wno-conversion
>
> We use the same compiler flags as px4 so not really appealing to change
> them. When you declare a constant, just put an f at the end and it will be
> a float constant instead of a double constant.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/PX4/Matrix/pull/26/files/3a1e1e3d2bdfaebd9efd87bdeb9d135dfe6d4453#r74969099>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AERqiYYlOV98KM2MpEo3Ew8mENDszSd3ks5qgeQ0gaJpZM4JZZwu>
> .
>